src/desktop/ui/fillsettings.ui

```diff
@@ -237,6 +237,9 @@
      <property name="prefix">
       <string>Expand: </string>
      </property>
+     <property name="minimum">
+       <number>-30</number>
+      </property>
```

Comment:
Dito to the above, negative values only make sense if the rest of #1188 is actually implemented.